THIRD-PARTY.txt

```diff
@@ -136,6 +136,7 @@ List of third-party dependencies grouped by their license type.
         * FindBugs-jsr305 (com.google.code.findbugs:jsr305:3.0.2 - http://findbugs.sourceforge.net/)
         * Google Android Annotations Library (com.google.android:annotations:4.1.1.4 - http://source.android.com/)
         * Graphite Integration for Metrics (io.dropwizard.metrics:metrics-graphite:3.2.6 - http://metrics.dropwizard.io/metrics-graphite/)
+        * Gson (com.google.code.gson:gson:2.8.9 - https://github.com/google/gson/gson)
```

Comment:
I think, that something is leaking `gson` here. Maybe we can fix to `2.9.0` ?

Reply:
Not sure what generates it. 

`mvn dependency:tree `

```
[INFO] org.apache.stormcrawler:stormcrawler-playwright:jar:3.1.0-SNAPSHOT
[INFO] +- com.microsoft.playwright:playwright:jar:1.43.0:compile
[INFO] |  +- com.google.code.gson:gson:jar:2.10.1:compile
[INFO] |  +- org.opentest4j:opentest4j:jar:1.3.0:compile
```

Maybe it came from using an older version of playwright but then it would be removed from THIRD PARTY when calling `mvn clean install` ?